fish_prompt.fish

```diff
@@ -665,6 +669,15 @@ function __bobthefish_prompt_user -S -d 'Display current user and hostname'
         echo -ns (whoami)
     end
 
+    if set -q display_sudo_user
```

Comment:
We need to update the condition for showing an extra space on line 695. It looks great if you have `theme_display_user` too, but not when it’s only `theme_display_sudo_user`.

Reply:
I remember having this condition but while playing around I must have deleted it... :man_facepalming:  
Anyway, I added it back in and it should work now as expected.